### PR TITLE
chore(release): update versions for RC.6

### PR DESCRIPTION
**Pull request by kara**

### CODE REVIEW

**Comment by hansl** [comment]
The system-config used in our tests probably need to be updated as well.

File: src/lib/button-toggle/package.json [hunk 1]
```diff
@@ -24,6 +24,6 @@
   "homepage": "https://github.com/angular/material2#readme",
   "peerDependencies": {
     "@angular2-material/core": "2.0.0-alpha.7-12",
```
Comment (jelbourn) [inline src/lib/button-toggle/package.json]: Need to update the version to alpha 8

Comment (jelbourn) [inline src/lib/button-toggle/package.json]: Also need to update the change log. Those two things can happen in a separate pr, though.

Comment (kara) [inline src/lib/button-toggle/package.json]: Yeah, I'll include in a separate PR

**Comment by kara** [comment]
Got a verbal LGTM from Jeremy

**Pull request merged**